src/index.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const db = require("../src/db");
+
+const app = express();
+const port = process.env.PORT || 4000;
+
+const cors = require("cors");
+const corsMiddleWare = cors();
+
+app.use(corsMiddleWare);
+
+const bodyParser = require("body-parser");
+const bodyParserMiddleWare = bodyParser.json();
```

Comment:
use express's body parser so you use one less dependency 
`const bodyParserMiddleWare = express.json();`